src/build_defs.bzl

```diff
@@ -232,12 +487,14 @@ def mozc_cc_win32_library(
     # The actual DLL name should be specified in the LIBRARY section of
     # win_def_file.
     # https://learn.microsoft.com/en-us/cpp/build/reference/library
-    cc_binary_target_name = name + "_fake.dll"
-    filegroup_target_name = name + "_lib"
+    cc_binary_target_intermediate_name = name + "_fake"
+    cc_binary_target_name = cc_binary_target_intermediate_name + ".dll"
+    filegroup_target_name = name + "_lib_group"
+    cc_binary_target_libname = cc_binary_target_name + ".if.dll"
```

Comment:
Unused?

Reply:
Ah, seems that you were talking only about `cc_binary_target_libname`, right? Then, right, `cc_binary_target_libname` is not used. Will update it shortly.

Reply:
Removed `cc_binary_target_libname`.